README.md

```diff
@@ -62,3 +62,14 @@ docker run --rm -v cms_esdata:/volume -v /tmp:/backup alpine tar -cjf /backup/cm
 ```
 docker run --rm -v cms_esdata:/volume -v /tmp:/backup alpine sh -c "rm -rf /volume/* /volume/..?* /volume/.[!.]* ; tar -C /volume/ -xjf /backup/cms_esdata.tar.bz2"
 ```
+
+# Populating database tables
+
+Download the following files from S3 into a `test_data` directory
```

Comment:
Any reason not to do the downloads in the load script?

Reply:
I could do that, but I'd want to refactor this into separate "full-data" load scripts and "test-data" load scripts that call a core load script.  Let me put something in place tomorrow.

Reply:
Feel free to merge this in the meantime.

Reply:
I split the db_load script into 3 scripts: one to load test data, one or download and load full data, and a shared core script that they both call. 

It was a good suggestion; it will make it much easier to update the production database tables from updated files in S3 in the future.